packages/react-instantsearch-dom-maps/src/__tests__/GoogleMaps.js

```diff
@@ -388,8 +382,8 @@ describe('GoogleMaps', () => {
 
       expect(mapInstance.fitBounds).toHaveBeenCalledTimes(0);
 
-      expect(mapInstance.setZoom).toHaveBeenCalledTimes(1); // cDM
-      expect(mapInstance.setCenter).toHaveBeenCalledTimes(1); // cDM
+      expect(mapInstance.setZoom).toHaveBeenCalledTimes(2); // cDM
```

Comment:
Since we store the context in the state, the component renders twice now.

Reply:
The number of calls should not change if we render the `Provider` at the right time.